shared/js/background/email-utils.es6.js

```diff
@@ -83,6 +90,64 @@ export const getAddresses = () => {
     }
 }
 
+function sendPixelRequest (pixelName, params) {
+    const randomNum = Math.ceil(Math.random() * 1e7)
+
+    const searchParams = new URLSearchParams(Object.entries(params))
+
+    const url = getURL(pixelName) + `?${randomNum}&${searchParams.toString()}`
+
+    load.url(url)
```

Comment:
@sammacbeth I don't think I agree, this just a few lines, the one in broken-site-report is much longer and more complicated. Is it worth introducing another abstraction? I think it might be better to wait until we have more examples, to figure out good interface. I don't think there's any maintainability risk here.